assets/js/blocks/classic-template/index.tsx

```diff
@@ -68,48 +96,20 @@ const Edit = ( {
 				className="wp-block-woocommerce-classic-template__placeholder"
 			>
 				<div className="wp-block-woocommerce-classic-template__placeholder-copy">
-					<p className="wp-block-woocommerce-classic-template__placeholder-warning">
-						<strong>
-							{ __(
-								'Do not remove this block!',
-								'woo-gutenberg-products-block'
-							) }
-						</strong>{ ' ' }
-						{ __(
-							'Removing this will cause unintended effects on your store.',
-							'woo-gutenberg-products-block'
-						) }
-					</p>
-					<p>
-						{ sprintf(
-							/* translators: %s is the template title */
-							__(
-								'This is a placeholder for the %s. In your store it will display the actual product image, title, price, etc. You can move this placeholder around and add more blocks around it to customize the template.',
-								'woo-gutenberg-products-block'
-							),
-							templateTitle
-						) }
-					</p>
+					<p>{ placeholderDescription }</p>
 				</div>
 				<div className="wp-block-woocommerce-classic-template__placeholder-wireframe">
-					{ isExperimentalBuild() && (
+					{ canConvert && (
```

Comment:
Let's wait for the final decision on the templates before removing the feature gating because if this PR is getting into Core, the conversion will work there. We can either wait for #8308 or add the feature gating back to merge this PR now (and remove the flag later in #8308).

Reply:
I think I'm ok with waiting, so both PRs are aligned in terms of a block structure 👍

Reply:
@Aljullu, can I get your opinion about this?

I think I'll bring back the feature flag for now:
- the auto-conversion is not enabled yet in #8308 
- compatibility layer has been disabled for now as it caused regression https://github.com/woocommerce/woocommerce-blocks/pull/8507

Actually seems like there's no space for opinions 😅 but it has to be hidden behind the feature flag yet and all three parts have to be enabled together.

Reply:
Good point. Yes, it makes sense to me. :+1:

Reply:
Added [in the commit](https://github.com/woocommerce/woocommerce-blocks/pull/8248/commits/462fa4a4058903e8bb2498f8949a97c0e2b7d1f4)